src/networks/starknet/constants.ts

```diff
@@ -1,89 +1,194 @@
+
+export const API_URL = 'http://localhost:3000';
 
 export const SUPPORTED_AUTHENTICATORS = {
-  '0x64cce9272197eba6353f5bbf060e097e516b411e66e83a9cf5910a08697df14': true
+  '0x7ea118e919c2d693f6c6d4643caae86814e8a7c06a77c33799e8d5f8a544a2': true,
+  '0x6584c1eacea3c9721242ea4a795cfd4d63be30943d9686a64bfedf04765cd5c': true,
+  '0xb321c09ee9851c125bd4213de71ebd03c07813556bae5d4700968df42ee476': true,
+  '0x53c66d5b61f7b7f8a3871908b16d6e199ed401b706fb042a006d53f97ec2958': true
 };
 
 export const SUPPORTED_STRATEGIES = {
-  '0x58623786b93d9b6ed1f83cec5c6fa6bea5f399d2795ee56a6123bdd83f5aa48': true,
-  '0xd1b81feff3095ca9517fdfc7427e742ce96f7ca8f3b2664a21b2fba552493b': true
+  '0x510d1e6d386a2adcfc6f2a57f80c4c4268baeccbd4a09334e843b17ce9225ee': true,
+  '0x619040eb54857252396d0bf337dc7a7f98182fa015c11578201105038106cb7': true
 };
 
-export const SUPPORTED_EXECUTORS = {
-  '0x21dda40770f4317582251cffd5a0202d6b223dc167e5c8db25dc887d11eba81': true
-};
+export const SUPPORTED_EXECUTORS = {};
+
+export const RELAYER_AUTHENTICATORS = {
+  '0xb321c09ee9851c125bd4213de71ebd03c07813556bae5d4700968df42ee476': 'starknet',
+  '0x6584c1eacea3c9721242ea4a795cfd4d63be30943d9686a64bfedf04765cd5c': 'evm'
+} as const;
 
 export const AUTHS = {
-  '0x5e1f273ca9a11f78bfb291cbe1b49294cf3c76dd48951e7ab7db6d9fb1e7d62': 'Vanilla',
-  '0x64cce9272197eba6353f5bbf060e097e516b411e66e83a9cf5910a08697df14': 'Ethereum signature',
-  '0x4112e7aef90c47058238ccb76bf79ad5188afdf366870015185e3c7468ccbd9':
-    'Ethereum signature session key',
-  '0x68a2d3c6d882ec0e2e94042556878d27e832a28e1308df04ad35fd8bae9ec6b': 'Ethereum transaction',
-  '0x64bb9fd620d7e4c5f5895329e8f1d3d5f485ccfb2b16345a1ca86658f24c9f6':
-    'Ethereum transaction session key',
-  '0x59283b509832027a386b3f419628a5b149e9d6462a6547c63e92bd4a09a7245': 'Starknet signature'
+  '0x7ea118e919c2d693f6c6d4643caae86814e8a7c06a77c33799e8d5f8a544a2': 'Vanilla',
+  '0x6584c1eacea3c9721242ea4a795cfd4d63be30943d9686a64bfedf04765cd5c': 'Ethereum signature',
+  '0x204546a6d59f757677506cb6e6b031dd0f4990613ce6e9212a2e76c67a7dc54': 'Ethereum transaction',
+  '0xb321c09ee9851c125bd4213de71ebd03c07813556bae5d4700968df42ee476': 'Starknet signature',
+  '0x53c66d5b61f7b7f8a3871908b16d6e199ed401b706fb042a006d53f97ec2958': 'Starknet transaction'
 };
 
-export const PROPOSAL_VALIDATIONS = {};
+export const PROPOSAL_VALIDATIONS = {
+  '0x38f034f17941669555fca61c43c67a517263aaaab833b26a1ab877a21c0bb6d': 'Voting power'
+};
 
 export const STRATEGIES = {
-  '0x58623786b93d9b6ed1f83cec5c6fa6bea5f399d2795ee56a6123bdd83f5aa48': 'Vanilla',
-  '0xd1b81feff3095ca9517fdfc7427e742ce96f7ca8f3b2664a21b2fba552493b': 'Single slot proof'
+  '0x510d1e6d386a2adcfc6f2a57f80c4c4268baeccbd4a09334e843b17ce9225ee': 'Vanilla',
+  '0x297fb0104d8be6c86f168bf1dcdc28b0625d2b06108c3c46194aa4415f2e2ec': 'Merkle whitelist',
+  '0x619040eb54857252396d0bf337dc7a7f98182fa015c11578201105038106cb7': 'Delegated ERC20 Token'
```

Comment:
This should have same name than "OpenZeppelin Votes", actually both "OpenZeppelin Votes" and "Delegated ERC20 Token" are not ideal name, we should change both the EVM and Starknet strategies names to "ERC-20 Votes (EIP-5805)"

Reply:
Also "ERC-20 Votes (EIP-5805)" should appear before "Delegated Comp Token", and "Delegated Comp Token" should be renamed to "ERC-20 Votes Comp"